website/client/components/settings/site.vue

```diff
@@ -240,6 +234,9 @@ export default {
     // @TODO: We may need to request the party here
     this.party = this.$store.state.party;
     this.newDayStart = this.user.preferences.dayStart;
+    this.usernameUpdates.username = this.user.auth.local.username || null;
+    this.displayName = this.user.profile.name;
```

Comment:
I think this property should be renamed to `temporaryDisplayName` since it seems to be only used in a form to change the display name